rudder-agent-thin/SOURCES/Makefile

```diff
@@ -1,79 +1,23 @@
-#####################################################################################
-# Copyright 2014 Normation SAS
-#####################################################################################
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, Version 3.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-#####################################################################################
-
-.DEFAULT_GOAL := localdepends
-
-RUDDER_VERSION_TO_PACKAGE = <put Rudder version or version-snapshot here>
-
-MAKE  := /usr/bin/make
-PATCH := /usr/bin/patch
-FIND  := /usr/bin/find
-
-localdepends: ./rudder-agent.make build-sources
-
-# We need to copy rudder-agent's Makefile to make the build the most self-contained possible
-rudder-agent.make: ../../rudder-agent/SOURCES/Makefile
-	cp -f ../../rudder-agent/SOURCES/Makefile ./rudder-agent.make
-
-fetch-sources: fetch-sources-stamp
-fetch-sources-stamp:
-	# Clean up rudder-agent sources first
-	cd ../../rudder-agent/SOURCES && make localclean
-	# Make sure the debian/changelog is pristine
-	cd ../../rudder-agent/debian && git checkout changelog
```

Comment:
No, the packaging will call it directly
And since I used the Makefile restart hack, it will be called directly on the patched rudder-agent copy.